tests/foreman/cli/test_satellitesync.py

```diff
@@ -244,21 +322,38 @@ def test_positive_export_import_cv(self):
 
         :steps:
 
-            1. Export whole CV version contents to a directory specified in
-               settings.
-            2. Copy exported contents to /var/www/html/pub/export directory.
-            3. Import these copied contents from some other org/satellite.
+            1. Export whole CV version contents to a directory
+            3. Import those contents from some other org/satellite.
 
         :expectedresults:
 
             1. Whole CV version contents has been exported to directory
-               specified in settings.
             2. All The exported contents has been imported in org/satellite.
 
-        :caseautomation: notautomated
-
         :CaseLevel: System
         """
+        ContentView.version_export({
+            'export-dir': '{}'.format(self.export_dir),
+            'id': self.exporting_cvv_id
+        })
+        exported_tar = '{0}/export-{1}.tar'.format(self.export_dir, self.exporting_cvv_id)
+        result = ssh.command("[ -f {0} ]".format(exported_tar))
```

Comment:
I'd probably do more assertions on the archive here than a simple "file exists?" check.
the tar might be empty or already missing some files and we can catch it early.
perhaps at least asserting the output of:
```python
# lists the contents of the tar archive
ssh.command("tar -t -f {0}".format(exported_tar)
```
might help in case of the test failure. it's a single command and makes the debugging much easier - what do you think?

Reply:
I will do that right away, my plan was to do that but just forgot to update.

Reply:
Sorry for last comment, but there would be a separate testcase which will cover the tar related expectations. This is just an end to end.

Reply:
Excuse me, but what is the prupose of the lengthy endtoend when you can see what happened after it failed in the end? Why would you like to continue executing the test if you can simply check that it will fail at this point?

Reply:
..or perhaps i understand the concept of end-to-ends wrong, I don't know :))

Reply:
Yeah, it's just like completing the entire scenario works or not. The last assertion of packages comparison confirms that export and import complete scenario works.

The exported tar will be exposed in detail in coming test.